packages/web-components/src/menu/menu.styles.ts

```diff
@@ -1,18 +1,35 @@
 import { css } from '@microsoft/fast-element';
-import {} from '../theme/design-tokens.js';
+import { display } from '../utils/index.js';
 
 /** Menu styles
  * @public
  */
 export const styles = css`
+  ${display('inline-block')}
+
   :host {
-    position: relative;
-    z-index: var(--z-index-menu, 1);
+    display: inline-block;
+  }
+
+  ::slotted([slot='trigger']) {
+    anchor-name: --menu-trigger;
+  }
+
+  ::slotted([popover]) {
+    inset-area: block-end span-inline-end;
+    margin: 0;
+    max-height: var(--menu-max-height, auto);
```

Comment:
It would be great to document this somewhere. Maybe add a note to include this when we update the docs?

Reply:
Let me make you a story for that. And I'll note the CSS property on the class.

Reply:
Documented in `menu.ts` and Storybook and I found a bug and fixed that.

Reply:
Thanks! And thanks for adding this to menu!